assets/airports/vecc.json

```diff
@@ -0,0 +1,606 @@
+		"icao": "NIBLU",
+		"name": "NIBLU",
+		"suffix": {"01L":"1", "01R":"2"},
+		"rwy": {
+			"01L": ["CC333"],
+			"01R": ["CC322"]
+		},
+		"transitions": {
+			"BOKEV": ["LEGOS", "CC110", "BOKEV"],
+			"KAKID": ["LEGOS", "KAKID"],
+			"SURUP": ["LEGOS", "KAKID", "SURUP"]
+		},	
+		"body": ["NIBLU"],
+		"draw": [
+			["_19R", "CC333", "NIBLU"], 
+			["_19L", "CC322", "NIBLU*"], 
+			["NIBLU", "LEGOS", "CC110", "BOKEV*"],
+			["NIBLU", "LEGOS", "KAKID*", "SURUP*"]
+		]
+	},
+	"PITAM": {
+		"icao": "PITAM",
+		"name": "PITAM",
+		"rwy": {
+			"01L": [],
+			"01R": []
+		},
+		"transitions": {
+			"AGODA": ["CC107", "AGODA"],
+			"TEPAL": ["CC106", "DUMKA"],
+			"ONOTO": ["CC107", "ONOTO"]
+		},	
+		"body": ["PITAM"],
+		"draw": [
+			["_19R", "PITAM"], 
+			["_19L", "PITAM*"], 
+			["PITAM", "CC107", "AGODA*"],
+			["PITAM", "CC106", "DUMKA*"],
+			["PITAM", "CC107", "ONOTO*"]
+		]
+	},
+	"GUMAS": {
+		"icao": "GUMAS",
+		"name": "GUMAS",
+		"rwy": {
+			"19L": [],
+			"19R": []
+		},
+		"transitions": {
+			"AVPOP": ["AVPOP"],
+			"DOPID": ["CC108", "DOPID"]
+		},	
+		"body": ["GUMAS"],
+		"draw": [
+			["_01R", "DOPID"], 
+			["_01L", "DOPID*"],
+			["DOPID", "AVPOP*"],
+			["DOPID", "CC108", "DOPID*"]
+		]
+	},
+	"KAPOL": {
+		"icao": "KAPOL",
+		"name": "KAPOL",
+		"rwy": {
+			"19L": [],
+			"19R": []
+		},
+		"transitions": {
+			"AGODA": ["CC107", "AGODA"],
+			"DUMKA": ["CC105", "DUMKA"],
+			"ONOTO": ["CC107", "ONOTO"],
+			"SAREK": ["CC105", "SAREK"]
+		},	
+		"body": ["RIKLA", "CC308", "KAPOL"],
+		"draw": [
+			["_01R", "KAPOL"], 
+			["_01L", "KAPOL*"],
+			["KAPOL", "CC107", "AGODA*"],
+			["KAPOL", "CC105", "DUMKA*"],
+			["KAPOL", "CC107", "ONOTO*"],
+			["KAPOL", "CC105", "SAREK*"]
+		]
+	},
+	"KESAR": {
+		"icao": "KESAR",
+		"name": "KESAR",
+		"rwy": {
+			"19L": [],
+			"19R": []
+		},
+		"transitions": {
+			"BOKEV": ["LEGOS", "CC110", "BOKEV"],
+			"KAKID": ["LEGOS", "KAKID"],
+			"SURUP": ["LEGOS", "KAKID", "SURUP"]
+		},	
+		"body": ["KESAR"],
+		"draw": [
+			["_01R", "KESAR"], 
+			["_01L", "KESAR*"],
+			["KESAR", "LEGOS", "KAKID*", "SURUP*"]
+		]
+	},
+	"RIKLA": {
+		"icao": "RIKLA",
+		"name": "RIKLA",
+		"rwy": {
+			"19L": [],
+			"19R": []
+		},
+		"transitions": {
+			"JJS":		["CC101", "JJS"],
+			"TEPAL": 	["CC201", "CC203", "CC103", "TEPAL"]
+		},	
+		"body": ["RIKLA"],
+		"draw": [
+			["_01R", "RIKLA"], 
+			["_01L", "RIKLA*"],
+			["RIKLA", "CC101", "JJS*"],
+			["RIKLA", "CC201", "CC203", "CC103", "TEPAL*"]
+		]
+	},
+	"SEGOL": {
+		"icao": "SEGOL",
+		"name": "SEGOL",
+		"rwy": {
+			"19L": [],
+			"19R": []
+		},
+		"transitions": {
+			"BEMAK": ["CC312", "BEMAK"],
+			"NOKAT": ["CC312", "NOKAT"],
+			"SUMAG": ["SUMAG"]
+		},	
+		"body": ["SEGOL"],
+		"draw": [
+			["_01R", "SEGOL"], 
+			["_01L", "SEGOL*"],
+			["SEGOL", "CC312", "BEMAK*"],
+			["SEGOL", "CC312", "NOKAT*"],
+			["SEGOL", "SUMAG*"]
+		]
+	}
+  },
+ 
+  "departures": {
+    "airlines": [
+		["xax",  10],
+		["aic", 190],
+		["llr",  30],
```

Comment:
All of the entries for the `llr` airline are causing errors in the console because there is no file for either the `LLR` or `SEJ` airlines.

![image](https://cloud.githubusercontent.com/assets/5103735/15486712/3e78d0bc-210d-11e6-8c7f-ca14b42292b9.png)

I do see that you included these airlines in a different pull request, #524. So once #524 is fixed and merged, I can continue looking at this one.